src/pages/guides/security/understanding_authentication.md

```diff
@@ -10,9 +10,9 @@ keywords:
 
 If you plan to work with Adobe APIs in your custom application, please follow this chapter to understand Adobe Authentication and the libraries that you can leverage. 
 
-Adobe services like Creative SDK, Photoshop, Adobe Analytics, etc. use the OAuth 2.0 protocol for authentication and authorization. Using Adobe OAuth 2.0, you can generate an access token which is used to make API calls from your web server or browser-based apps. You can learn more about it at [OAuth 2.0 Authentication and Authorization](/authentication/auth-methods#!AdobeDocs/adobeio-auth/master/OAuth/OAuth.md).
+Adobe services like Creative SDK, Photoshop, Adobe Analytics, etc. use the OAuth 2.0 protocol for authentication and authorization. Using Adobe OAuth 2.0, you can generate an access token which is used to make API calls from your web server or browser-based apps. You can learn more about it at [OAuth 2.0 Authentication and Authorization](https://developer.adobe.com/developer-console/docs/guides/authentication/OAuth/).
```

Comment:
`/developer-console/docs/guides/authentication/OAuth/` should work. So basically remove `https://developer.adobe.com` to make sure it also works for staging

Reply:
thanks! updated